src/routes/+page.svelte

```diff
@@ -193,26 +196,32 @@
 			<h5>{$_('plannedDeposits')}</h5>
 			<a href={routes.DEPOSIT()}>+</a>
 		</div>
-		{#each detailStore.deposits as deposit, i}
-			<div>
-				{JSON.stringify(deposit)}<a href={routes.DEPOSIT(i)}>{$_('edit')}</a><button
-					onclick={() => detailStore.removeDeposit(i)}>{$_('delete')}</button
-				>
-			</div>
-		{/each}
+		<div class="grid">
+			{#each detailStore.deposits as deposit, i}
+				<Operation operation={deposit} currency={detailStore.currency}>
+					<div class="edit"><a href={routes.DEPOSIT(i)}><Edit size={24} /></a></div>
```

Comment:
This should also be a ghost button which however uses `a` element instead of `button` element.